### ISSUE TITLE
Produce warning when [max-]height:0 is used with a CSS animation or transition

### ISSUE THREAD

**Issue by weboverhauls** [CSSLint/csslint#363]
A warning should be generated when height:0 or max-height:0 is used to hide content with a CSS animation or transition.

If hiding content in this manner, the content may not be hidden to all users as intended, particularly to screen reader users.

The following patterns are considered warnings:

``` css
#myContainer {
  max-height: 0;
  transition: max-height 1s ease 0s;
}

@keyframes hide {
  from {
    max-height: 5em;
  }
  to {
    max-height: 0;
  }
}
```

**Comment by mgifford** [comment]
What is the status of this?
